### ISSUE TITLE
update README file - compatibility section 

### ISSUE THREAD

**Issue by researcher8173** [asLody/whale#63]
The title of one of the commits is "Feature: add support for Android R"
does its mean that the project compatible with the newer Android OSs?

**Comment by vegedreamgagnoa** [comment]
@researcher8173 Yeah, it supports Android R. But Android S, S_L (32), and tiramisu are not supported! so the answer, no, it does not compatible with newer Android OSes.
